src/api-routes/persons/{osuId}.js

```diff
@@ -0,0 +1,24 @@
+import { errorHandler, errorBuilder } from 'errors/errors';
+import { getPersonById } from '../../db/oracledb/persons-dao';
+
+/**
+ * Get person by ID
+ *
+ * @param {object} req request
+ * @param {object} res response
+ * @returns {Promise<object>} response
+ */
+const get = async (req, res) => {
+  try {
+    const { params: binds } = req;
```

Comment:
Almost all the rest APIs directly fetch the ID as a string like this: https://github.com/osu-mist/identities-api/blob/83d0112d7624a393b7232711d89408ce1d049dc7/src/api-routes/identities/%7BosuId%7D.js#L11

But actually I'm fine either way, so it's your call :)

Reply:
I'll keep it consistent